packages/example/src/index.tsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import { ChainId, DAppProvider } from '@usedapp/core'
+import { ChainId, DAppProvider, Config } from '@usedapp/core'
 import { App } from './App'
-import type { Config } from '@usedapp/core/dist/src/model/Config'
 
-const config: Config = {
-  readOnlyChain: ChainId.Mainnet,
+const config: Partial<Config> = {
```

Comment:
I don't like that the user has to change our type with `Partial` here.
Also it prevents us from adding really required fields (without a default value) in the future.

How about making the fields in `Config` optional, and maybe having e.g. `ConfigInternal` with fields not-optional (after supplementing with `DEFAULT_VALUE`)?

Reply:
✅